plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx

```diff
@@ -24,6 +24,7 @@ import Typography from '@material-ui/core/Typography';
 import { makeStyles } from '@material-ui/core/styles';
 import { Notification } from '@backstage/plugin-notifications-common';
 import { useConfirm } from 'material-ui-confirm';
+import BroadcastIcon from '@material-ui/icons/Cast'; /* TODO: Change to Podcasts or CellTower once updated */
```

Comment:
Have you considered the `RssFeedIcon`?
![image](https://github.com/user-attachments/assets/e5f3ca53-18e1-44bf-9c58-2d05e03c8c47)

Reply:
Thanks, looks better.